yarn-project/circuit-types/src/simulation_error.ts

```diff
@@ -175,7 +176,7 @@ export class SimulationError extends Error {
     const stackLines: string[] = [
       ...functionCallStack.map(failingFunction => {
         return `at ${failingFunction.contractName ?? failingFunction.contractAddress.toString()}.${
-          failingFunction.functionName ?? failingFunction.functionSelector.toString()
+          failingFunction.functionName ?? failingFunction.functionSelector?.toString() ?? ''
```

Comment:
maybe better '<unknown>' instead of '', to signal that the function is unknown. Otherwise you just get `0x9182.` (note the `.`)